### PR TITLE
[Breaking] Switch moonsense.models to be generated from protocol buffer definitions

### PR DESCRIPTION
**Pull request by andreisavu**
This simplifies ongoing maintenance and introduces the set of objects needed
to implement webhook handlers that will receive the bundle payload in the request
body.

There are changes in the signature of the Client class methods. All examples and
tests have been updated as needed.

### CODE REVIEW

**Comment by andreisavu** [comment]
Added a commit with a webhook handler example that does simple gesture processing.

File: consumer_example.py [hunk 1]
```diff
@@ -39,13 +39,15 @@
...
+    bundles = list(client.read_chunk(session.session_id, chunk.chunk_id))
 
-    client.create_card(
-        session.session_id,
-        "Python Consumer Example",
-        f"Chunk {chunk.chunk_id} has {len(bundles)} bundles",
-    )
+    logging.info(f"Chunk {chunk.chunk_id} has {len(bundles)} bundles")
+
+    # client.create_card(
```
Comment (jnsimon) [inline consumer_example.py:46]: can the comment block be removed?

File: moonsense/pd.py [hunk 2]
```diff
@@ -29,7 +28,7 @@ def read_accelerometer(session_file) -> pd.DataFrame:
     df = pd.read_json(session_file, lines=True)
     accelerometer_data = functools.reduce(
         operator.iconcat,
-        df["bundle"].apply(pd.Series)["accelerometer_data"].tolist(),
+        df["bundle"].apply(pd.Series)["accelerometer_data"].dropna().tolist(),
```
Comment (jnsimon) [inline moonsense/pd.py:31]: why is the dropna() needed? Are the bundles returning empty data?

**Review by jnsimon — APPROVED** [review]
looks good. This will make it much easier to incorporate updates

File: consumer_example.py (hunk 1 above)
Comment (andreisavu) [reply consumer_example.py:46]: I want to keep it in as a way to show how a call to create a card would look like. this is only relevant for the Recorder.

File: moonsense/pd.py (hunk 2 above)
Comment (andreisavu) [reply moonsense/pd.py:31]: I got to see an instance of that. Haven't spent time debugging the root cause yet. We will need to keep an eye on it in the future.

**Pull request merged**